src/proj_4D_api.c

```diff
@@ -795,7 +758,7 @@ PJ_INIT_INFO proj_init_info(const char *initname){
     metadata is missing, the value is set to "Unknown".
 
 ******************************************************************************/
-    int file_found, def_found=0;
+    int file_found;
     char param[80], key[74];
```

Comment:
It would be better to have 80 and 74 be symbolic constants.  Out of scope for this PR, but just want to mention this.